### PR TITLE
[add] website_menu_permission

### PR DESCRIPTION
**Pull request by simahawk**
Replace or make https://github.com/OCA/website/pull/324 dependent on this.

**TODO**

- [ ] finish README
- [ ] move tests setup to class setup
- [ ] move shortcut flags to website_menu_by_user_status???
- [ ] not required but nice to have: advanced perm form

### CODE REVIEW

**Comment by GillesTephaneMeyomesse** [comment]
@simahawk thanks i'll check this asap

**Comment by GillesTephaneMeyomesse** [comment]
Hi @simahawk Would it be possible for me to integrate your developments on my branch and add you as a contributor ?

**Comment by simahawk** [comment]
@GillesTephaneMeyomesse what do you want to do? :)
BTW we can chat on hangout or IRC to coordinate and speed up the process.

**Comment by hbrunn** [comment]
looks good so far. If you rely on groups anyways for logged in / not logged in, why have special treatment for this in the first place?

You can add the many2many directly to the treeview, that works.

**Comment by simahawk** [comment]
@hbrunn is just to glue it w/ std behavior of `website_menu_by_user_status` and actually my TODO comment says "move shortcut flags to website_menu_by_user_status" ;)
So, we can strip that out.

**Comment by rafaelbn** [comment]
Please @simahawk ping me and @cristinamartinrod when you finish the TODO we will review with pleasure

**Comment by simahawk** [comment]
@rafaelbn during OCA CS @hbrunn and I agreed on the fact that we should this module as a base for all limitations on menu items. We should just base menu_by_user_status on this and move there the easy setting via flags. I'll try to get back on this soon.

File: website_menu_permission/views/website_views.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<odoo>
+  <record id="add_menu_permission" model="ir.ui.view">
+    <field name="name">website_menu_permission: show flags</field>
+    <field name="model">website.menu</field>
+    <field name="inherit_id" ref="website.menu_tree"/>
+    <field name="arch" type="xml">
+      <field name="new_window" position="after">
+        <field name="user_logged"/>
+        <field name="user_not_logged"/>
+        <!-- TODO: add action to edit "advanced permissions" and play w/ groups -->
```
Comment (thomaspaulb) [inline website_menu_permission/views/website_views.xml:11]: Could we just add `<field name="groups_id" widget="many2many_tags"/>` here and be done with it

**Pull request closed without merging**